packages/wallets/src/features/cfd/screens/ServerMaintenance/ServerMaintenance.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { WalletButton, WalletText } from '../../../../components/Base';
+import { useModal } from '../../../../components/ModalProvider';
+import useDevice from '../../../../hooks/useDevice';
+import { CFD_PLATFORMS } from '../../constants';
+import './ServerMaintenance.scss';
+
+const ServerMaintenance: React.FC = () => {
+    const { hide } = useModal();
+    const { isMobile } = useDevice();
+
+    const getMaintenanceTime = (platform: string) => {
```

Comment:
Server maintenance should come from the API, it might be dynamic and it is not right to keep it in the FE code.